asciidoc.py

```diff
@@ -2265,7 +2265,8 @@ def gen_id(title):
         base_id = re.sub(r'\W+', '_', title).strip('_').lower()
         if 'ascii-ids' in document.attributes:
             # Replace non-ASCII characters with ASCII equivalents.
-            base_id = unicodedata.normalize('NFKD', base_id).encode('ascii', 'ignore').decode('ascii')
+            from trans import trans
+            base_id = trans(base_id)
```

Comment:
```suggestion
            try:
                from trans import trans
                base_id = trans(base_id)
            except ImportError:
                base_id = unicodedata.normalize('NFKD', base_id).encode('ascii', 'ignore').decode('ascii')
```

Until we move a proper setup.py, I do not want to make any python module mandatory. This would also give the ability to deprecate how ids are currently handled for 9.0.0 and move totally to trans for 10.0.0.